### PR TITLE
Fixing problems in win in various apps

### PR DESCRIPTION
**Pull request by roigcarlo**
Some win problems in win for the release. We shall merge this to the master after so don't delete the branch

### CODE REVIEW

File: applications/MeshMovingApplication/CMakeLists.txt [hunk 1]
```diff
@@ -25,6 +25,7 @@ endif(${KRATOS_BUILD_TESTING} MATCHES ON)
 ## define library Kratos which defines the basic python interface
 pybind11_add_module(KratosMeshMovingApplication MODULE THIN_LTO ${KRATOS_MESH_MOVING_APPLICATION_SOURCES} ${KRATOS_MESH_MOVING_TESTING_SOURCES})
 target_link_libraries(KratosMeshMovingApplication PRIVATE KratosCore )
+set_target_properties(KratosMeshMovingApplication PROPERTIES COMPILE_DEFINITIONS "MESH_MOVING_APPLICATION=EXPORT,API")
```
Comment (philbucher) [inline applications/MeshMovingApplication/CMakeLists.txt:28]: @longchentum could you try if this works for you?

see #2146

**Comment by roigcarlo** [comment]
Can someone approve this pls? I need it to generate the release betas

**Review by jcotela — APPROVED** [review]

**Comment by roigcarlo** [comment]
thx

**Comment by philbucher** [comment]
is this branch still needed?

**Comment by roigcarlo** [comment]
Nope, deleted

**Pull request merged**